test/XRP/fakes/fake-xrp-protobufs.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-statements -- long functions are fine here */
```

Comment:
This is a weird request, but would you mind renaming this file to "fake-protobufs.ts" without the "xrp" ? I know it's probably called this in `xpring-js`, but I feel like our indiscriminate use of "xrp" in naming things is a relic of when the design of the SDK was going to be limited to only XRP-related functionality.  In most places it's used, we should really have been using either "Xrpl" to indicate the XRP Ledger, or nothing at all.  So just a philosophy we should think about going forward.